cmd/kpod/stop.go

```diff
@@ -0,0 +1,83 @@
+package main
+
+import (
+	"fmt"
+	"github.com/Sirupsen/logrus"
+	"github.com/containers/storage/pkg/reexec"
+	"github.com/kubernetes-incubator/cri-o/server"
+	"github.com/pkg/errors"
+	"github.com/urfave/cli"
+	"golang.org/x/net/context"
+	pb "k8s.io/kubernetes/pkg/kubelet/api/v1alpha1/runtime"
+	"os"
+)
+
+const crioConfigPath = "/etc/crio/crio.conf"
+
+var stopTimeout int64
+var configPath string
+
+var (
+	stopFlags = []cli.Flag{
+		cli.Int64Flag{
+			Name:        "timeout, t",
+			Usage:       "Seconds to wait for stop before killing it (default 10)",
```

Comment:
I don't think so, in this instance you could read it that you're killing the stop process instead of the container.  If you replace "it" here with "the container", that would work for me.